### PR TITLE
add variant gmock to googletest

### PR DESCRIPTION
**Pull request by healther**

### CODE REVIEW

**Comment by healther** [comment]
@alalazo  (How) Can I retrigger Travis?

**Comment by alalazo** [comment]
I don't think you can retrigger single jobs, as that possibly requires the status of "collaborator". Closing and re-opening a PR like you did is fine, though it retriggers all the builds.

File: var/spack/repos/builtin/packages/googletest/package.py [hunk 1]
```diff
@@ -34,11 +34,17 @@ class Googletest(CMakePackage):
     version('1.7.0', '5eaf03ed925a47b37c8e1d559eb19bc4')
     version('1.6.0', '90407321648ab25b067fcd798caf8c78')
 
+    variant('gmock', default=False, description='Build with gmock')
+
     def cmake_args(self):
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/googletest/package.py]: Can you put a:
```python
conflict('+gmock', when='@:1.7.0')
```
?

Comment (healther) [reply var/spack/repos/builtin/packages/googletest/package.py]: done

**Comment by healther** [comment]
I'm unsure what this Travis error is trying to tell me

**Comment by scheibelp** [comment]
> I'm unsure what this Travis error is trying to tell me

This is probably due to the travis mac builds going down for a few days. I'm going to re-reopen it.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**